src/brian.py

```diff
@@ -63,12 +63,19 @@ def router(options):
             try:
                 router.send_multipart(frames)
             except zmq.ZMQError as e:
-                log.debug(f"unable to send frame: receiver->sender",
-                          sender=frames[1],
-                          receiver=frames[0],
-                          error=str(e))
+                # Check if message was intended for realtime, and drop the message if so
+                if e.errno == zmq.EHOSTUNREACH and frames[1].decode('utf-8').endswith("RT_IDEN"):
```

Comment:
Could we make the `endswith()` ambiguous by grabbing `dw_to_rt_identity` from the options file? 

In case we ever change the zmq identity strings

Reply:
That's a good idea